apps/desktop/src/lib/components/Board.svelte

```diff
@@ -11,25 +11,33 @@
 	import { VirtualBranchService } from '$lib/vbranches/virtualBranch';
 	import { getContext } from '@gitbutler/shared/context';
 	import { flip } from 'svelte/animate';
+	import type { VirtualBranch } from '$lib/vbranches/types';
 
 	const vbranchService = getContext(VirtualBranchService);
 	const branchController = getContext(BranchController);
 	const error = vbranchService.error;
 	const branches = vbranchService.branches;
 
-	let dragged: HTMLDivElement | undefined;
-	let dropZone: HTMLDivElement;
+	let dragged = $state<HTMLDivElement>();
+	let dropZone = $state<HTMLDivElement>();
 
-	let dragHandle: any;
-	let clone: any;
-	$: if ($error) {
-		$showHistoryView = true;
-	}
-	$: sortedBranches = $branches?.sort((a, b) => a.order - b.order) || [];
+	let dragHandle: any = $state();
+	let clone: any = $state();
+	$effect(() => {
+		if ($error) {
+			$showHistoryView = true;
+		}
+	});
+
+	let sortedBranches = $state<VirtualBranch[]>();
+
+	$effect(() => {
+		sortedBranches = $branches?.sort((a, b) => a.order - b.order) || [];
+	});
```

Comment:
Looks like it could be derived.

Reply:
This gets reassigned a little further down so unfortunately can't use `derived`